src/vendor_upstream/core/shallowEqual.js

```diff
@@ -18,7 +18,7 @@
  * when any key has values which are not strictly equal between the arguments.
  * Returns true when the values of all keys are strictly equal.
  */
-function shallowEqual(objA: mixed, objB: mixed): boolean {
+function shallowEqual(objA: mixed, objB: mixed, ignoredKeys): boolean {
```

Comment:
can we move this method outside of vendor_upstream so if we switch back to fbjs it isn't lost